webapp/app/forms/steps/unlock_code_activation_steps.py

```diff
@@ -29,8 +29,16 @@ def __init__(self, **kwargs):
             intro=_('form.unlock-code-activation.input-intro'),
             form=self.Form,
             **kwargs,
-            header_title=_('form.unlock-code-activation.header-title'),
-            template='basis/form_standard.html')
+            header_title=_('form.unlock-code-activation.header-title'))
+
+    def render(self, data, render_info):
+        render_info.form.first_field = next(iter(render_info.form))
+        return render_template(
+            'unlock_code/login.html',
```

Comment:
Couldn't you have also just changed the template in line 33?

Reply:
Very true and I was considering it, but I anticipate to substantially change what is passed into the template and so I chose to override the render method already, knowing I'll be needing it very soon. Maybe YAGNI, but I doubt it.

Reply:
Ok 👍